app.js

```diff
@@ -27,3 +27,320 @@ hamburgerCloseConst.addEventListener('click', closeMenu);
   item.addEventListener('click', handlePageLinkClick);
 });
+
+// popup
+
+const projects = [
+  {
+    name: 'Professional Art Printing Data',
+    description:
+      ' A daily selection of privately personalized reads  Lorem Ipsum is simply dummy text of the printing and typesetting industry ',
+    image: 'screen.png',
+    technologies: ['HTML', 'Bootstrap', 'Ruby'],
+    liveLink: 'https://google.com',
+    sourceLink: 'https://google.com',
+  },
+  {
+    name: 'Data Dashboard Healthcare',
+    description:
+      ' A daily selection of privately personalized reads  Lorem Ipsum is simply dummy text of the printing and typesetting industry ',
+    image: 'screen.png',
+    technologies: ['HTML', 'Bootstrap', 'Ruby'],
+    liveLink: 'https://google.com',
+    sourceLink: 'https://google.com',
+  },
+  {
+    name: 'Website Portfolio',
+    description:
+      ' A daily selection of privately personalized reads  Lorem Ipsum is simply dummy text of the printing and typesetting industry ',
+    image: 'screen.png',
+    technologies: ['HTML', 'Bootstrap', 'Ruby'],
+    liveLink: 'https://google.com',
+    sourceLink: 'https://google.com',
+  },
+  {
+    name: 'Professional Art Printing Data',
+    description:
+      ' A daily selection of privately personalized reads  Lorem Ipsum is simply dummy text of the printing and typesetting industry ',
+    image: 'screen.png',
+    technologies: ['HTML', 'Bootstrap', 'Ruby'],
+    liveLink: 'https://google.com',
+    sourceLink: 'https://google.com',
+  },
+  {
+    name: 'Data Dashboard Healthcare',
+    description:
+      ' A daily selection of privately personalized reads  Lorem Ipsum is simply dummy text of the printing and typesetting industry ',
+    image: 'screen.png',
+    technologies: ['HTML', 'Bootstrap', 'Ruby'],
+    liveLink: 'https://google.com',
+    sourceLink: 'https://google.com',
+  },
+  {
+    name: 'Website Portfolio',
+    description:
+      ' A daily selection of privately personalized reads  Lorem Ipsum is simply dummy text of the printing and typesetting industry ',
+    image: 'screen.png',
+    technologies: ['HTML', 'Bootstrap', 'Ruby'],
+    liveLink: 'https://google.com',
+    sourceLink: 'https://google.com',
+  },
+];
+
+let mainDivOne;
+
+function displayPopup(projectIndex) {
+  const project = projects[projectIndex];
+
+  if (!mainDivOne) {
+    mainDivOne = document.createElement('div');
+    mainDivOne.setAttribute('class', 'mainDiv');
+    document.body.appendChild(mainDivOne);
+  } else {
+    mainDivOne.innerHTML = '';
+  }
+
+  const iconImg = document.createElement('img');
+  iconImg.setAttribute('src', 'close.png');
+  iconImg.setAttribute('class', 'iconOnTop');
+
+  mainDivOne.appendChild(iconImg);
+
+  const mainDivTwo = document.createElement('div');
+  mainDivTwo.setAttribute('class', 'firstDiv');
+  mainDivOne.appendChild(mainDivTwo);
+
+  const mainText = document.createElement('h1');
+  mainText.textContent = project.name;
```

Comment:
Nice job implementing the popup functionalities and dynamically generating your project section as required😉. but let's try to make the popup match the given design as this is a requirement for this milestone👍🏼.

yours

<img width="1325" alt="Screenshot 2023-07-26 at 16 19 27" src="https://github.com/bhushan354/My-Portfolio/assets/64643995/f3f5d82f-782a-4800-b70b-557f6dcc3b5f">

design
<img width="541" alt="Screenshot 2023-07-26 at 16 47 26" src="https://github.com/bhushan354/My-Portfolio/assets/64643995/b807189e-ea61-46da-b309-cacc9d78d8e0">

take note of the highlighted elements and add the necessary spacings around them👍🏼. for instance, you can see the cancel button is not aligned properly with the heading like it is in the design, and also the button icons are not well positioned😉. please take note of all the highlighted elements and make the necessary change.

NOTE: the same thing applies to your mobile screen as there are also some deviations from the design. Kindly take a look at the design and ensure everything matches both on mobile and desktop😉.